packages/integration-tests/index.js

```diff
@@ -271,6 +271,43 @@ async function addRules(config, dataDirectory) {
   return Promise.all(promises).then((rs) => rs.length);
 }
 
+/**
+ * deletes a rule by name
+ *
+ * @param {string} name - name of the rule to delete.
+ * @returns {Promise.<dynamodbDocClient.delete>} - superclass delete promise
+ */
+async function _deleteOneRule(name) {
+  const r = new Rule();
+  return r.get({ name: name }).then((item) => r.delete(item));
+}
+
+
+/**
+ * returns a list of rule objects
+ *
+ * @param {string} stackName - Cloud formation stack name
+ * @param {string} bucketName - S3 internal bucket name
+ * @param {string} rulesDirectory - The directory continaing rules json files
+ * @returns {list} - list of rules found in rulesDirectory
+ */
+async function rulesList(stackName, bucketName, rulesDirectory) {
+  return setupSeedData(stackName, bucketName, rulesDirectory);
+}
+
+/**
+ *
+ * @param {string} stackName - Cloud formation stack name
+ * @param {string} bucketName - S3 internal bucket name
+ * @param {Array} rules - List of rules objects to delete
+ * @returns {Promise.<integer>} - Number of rules deleted
+ */
+async function deleteRules(stackName, bucketName, rules) {
+  setProcessEnvironment(stackName, bucketName);
+  const promises = rules.map((rule) => limit(() => _deleteOneRule(rule.name)));
```

Comment:
Is there a reason we can't use https://github.com/nasa/cumulus/blob/master/packages/api/models/rules.js#L35-L50 ?

Reply:
Do you mean get rid of _deleteOneRule() helper?  I think that's what that is doing. 
https://github.com/nasa/cumulus/blob/CUMULUS-718-kinesis-integration-tests/packages/integration-tests/index.js#L274-L283

Reply:
I see, I thought these were already rules objects. My mistake, you can ignore this comment.